cloudify_aws/s3/object.py

```diff
@@ -0,0 +1,171 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+# Stdlib imports
+
+# Third party imports
+
+# Cloudify imports
+from cloudify import ctx
+from cloudify.decorators import operation
+from cloudify.exceptions import NonRecoverableError
+
+# This package imports
+from cloudify_aws import constants
+from cloudify_aws.base import AwsBaseNode
+from cloudify_aws.connection import EC2ConnectionClient
+
+
+@operation
+def create(args=None, **_):
+    return Object().created(args)
+
+
+@operation
+def delete(args=None, **_):
+    return Object().deleted(args)
+
+
+class Object(AwsBaseNode):
```

Comment:
That goes against the pattern that we have for all the other entities, and the real problem with 'object' is actually the lower case one. I did consider calling the file s3_object, but if it's enough of a problem then just importing s3 clears up the name issue.